### ISSUE TITLE
Schema fixing: gridinfo should link to datacollection not datacollectiongroup

### ISSUE THREAD

**Issue by stufisher** [ispyb/ispyb-database-documentation#35]
GridInfo currently links to DataCollectionGroup, and not DataCollection. This means that a datacollectiongroup cannot contain multiple grid scans. I realise at ESRF you group these things using workflow, but we should not make these sort of assumptions.

**Comment by KarlLevik** [comment]
Agreed at meeting today: we will have both FKs for a while, then delete the DCGID at some point in the future.

**Comment by antolinos** [comment]
Right
